### PR TITLE
Moderator slack bot

### LINKED ISSUE #34: Create simple demo of slack bot

### PR DESCRIPTION
**Pull request by hzlmn**
## What do these changes do?
Closes #34 

## Checklist

- [ ] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] If you provide code modification, please add yourself to `CONTRIBUTORS.txt`
  * The format is `<Name> <Surname>`.
  * Please keep alphabetical order, the file is sorted by names. 
- [ ] Add a new news fragment into the `CHANGES` folder
  * name it `<issue_id>.<type>` (e.g. `588.bugfix`)
  * if you don't have an `issue_id` change it to the pr id after creating the PR
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: `Fix issue with non-ascii contents in doctest text files.`

### CODE REVIEW

File: demos/moderator_bot/tests/test_handlers.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+async def test_listen_challenge(client):
+    resp = await client.post("/listen", json={"challenge": "challenge"})
+    result = await resp.text()
+    assert "challenge" in result
+
+
+async def test_list_message(client):
+    await client.post(
```
Comment (gyermolenko) [inline demos/moderator_bot/tests/test_handlers.py]: can you explain this line please?

**Comment by gyermolenko** [comment]
Could you add existing test to [makefile](https://github.com/aio-libs/aiohttp-demos/blob/master/Makefile#L8) please so it will run in CI.

I'm not sure if it requires changes in [travis file ](https://github.com/aio-libs/aiohttp-demos/blob/master/.travis.yml#L16) also  since you run the app as a module.

File: demos/moderator_bot/tests/test_handlers.py (hunk 1 above)
Comment (hzlmn) [reply demos/moderator_bot/tests/test_handlers.py]: No need to check response here as we just call slack api under the hood. But i will add request mocking to actually check that it was sent.

**Comment by gyermolenko** [comment]
I did a couple of minor updates before merge, hope you don't mind. 
E.g. 10 Mb for a picture that will be shown in the main readme is too much.

Thanks again for contribution!

**Pull request merged**